disnake/interactions/base.py

```diff
@@ -95,9 +96,10 @@
 MISSING: Any = utils.MISSING
 
 T = TypeVar("T")
+BotT = TypeVar("BotT", bound="Client")
```

Comment:
small nit, this would be better as `ClientT` instead of `BotT` here, as long as we're not in ext.commands